tests/lib.rs

```diff
@@ -2,6 +2,9 @@ extern crate bson;
 
 #[macro_use(add_to_doc, doc)]
 extern crate mongodb;
+extern crate rustc_serialize;
+extern crate nalgebra as na;
```

Comment:
Okay, I was just following the example in the README:

---
The preferred way to use nalgebra is to import types and traits explicitly, and call free-functions using the na:: prefix:

```
extern crate nalgebra as na;
use na::{Vec3, Rot3, Rotation};

fn main() {
    let     a = Vec3::new(1.0f64, 1.0, 1.0);
    let mut b = Rot3::new(na::zero());

    b.append_rotation_mut(&a);

    assert!(na::approx_eq(&na::rotation(&b), &a));
}
```
---
https://github.com/sebcrozet/nalgebra